db/migrations/1640181397482-addSyncGeometryType.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class addSyncGeometryType1640181397482 implements MigrationInterface {
+  name = 'addSyncGeometryType1640181397482';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`CREATE TYPE "osm_sync_tracker"."sync_geometry_type_enum" AS ENUM('point', 'line', 'poly')`);
```

Comment:
don't use shortened names. use full names (linestring, polygon).

Reply:
I'm seeing it just now, after I've just changed `poly` to `polygon`. :fearful:

Reply:
done.